src/wallet/wallet2.cpp

```diff
@@ -3548,10 +3603,137 @@ std::vector<wallet2::pending_tx> wallet2::create_transactions(std::vector<crypto
   }
 }
 
+void wallet2::light_wallet_get_outs(std::vector<std::vector<tools::wallet2::get_outs_entry>> &outs, const std::list<size_t> &selected_transfers, size_t fake_outputs_count) {
```

Comment:
Note that I haven't been able to test this with non-rct outputs cause i don't own any.